src/providers/paypro/paypro.ts

```diff
@@ -12,11 +13,25 @@ export class PayproProvider {
     private logger: Logger,
     private bwcProvider: BwcProvider,
     private currencyProvider: CurrencyProvider,
-    private onGoingProcessProvider: OnGoingProcessProvider
+    private onGoingProcessProvider: OnGoingProcessProvider,
+    private actionSheetProvider: ActionSheetProvider
   ) {
     this.logger.debug('PayproProvider initialized');
   }
 
+  private showErrorInfoSheet(
```

Comment:
To use `default-error`, please use `errorsProvider`

Reply:
Fixed.